.github/workflows/python-test.yml

```diff
@@ -30,7 +30,11 @@ jobs:
         pip install -e .
     - name: Run unit tests
       run: |
-        pytest --cov=. --cov-report=xml
+        pytest --cov=. --cov-report=xml -k "generic"
+        pytest --cov=. --cov-report=xml --cov-append -k "classification"
+        pytest --cov=. --cov-report=xml --cov-append -k "regression"
+        pytest --cov=. --cov-report=xml --cov-append -k "segmentation and not transunet"
+        pytest --cov=. --cov-report=xml --cov-append -k "segmentation and transunet"
```

Comment:
Oh, I see what you mean. Sure, I'll do that. I'll add a few more descriptors to the test names to make pinpointing easier as well.

Reply:
OK, in the interests of getting TransUNet in asap, I'm just going to add the CI test step splitting as a base for now and look at putting in a separate PR for the finer separation. Since right now, all the general-case training/inference tests use loops instead of parametrizing for the model, maybe I can change the way it's organized so that CI will be able to show what model types are failing without needing to give every model type separate test functions.